cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/common/DBUtils.java

```diff
@@ -88,10 +88,10 @@ private static void shutDownMySQLAbandonedConnectionCleanupThread(ClassLoader cl
       Class<?> mysqlCleanupThreadClass = classLoader.loadClass("com.mysql.jdbc.AbandonedConnectionCleanupThread");
       Method shutdownMethod = mysqlCleanupThreadClass.getMethod("shutdown");
       shutdownMethod.invoke(null);
-      LOG.info("Successfully shutdown MySQL connection cleanup thread.");
+      LOG.info("Successfully shutdown RDBMS connection cleanup thread.");
     } catch (Throwable e) {
       // cleanup failed, ignoring silently
-      LOG.warn("Failed to shutdown MySQL connection cleanup thread. Ignoring.");
+      LOG.warn("Failed to shutdown RDBMS connection cleanup thread. Ignoring.");
```

Comment:
The old LOG message should be improved since this might be confusing if someone is using RDBMS other than MySQL.